### DISCUSSION TITLE
error in Filament\Resources\Resource::authorize() when upgrading filament v3

### DISCUSSION THREAD

**Discussion by anindyoy** [filamentphp/filament#8318] [Help]
Symfony\Component\ErrorHandler\Error\FatalError

  Cannot make static method Filament\Resources\Resource::authorize() non static in class Illuminate\Foundation\Auth\Access\AuthorizesRequests

i've got above error when running composer update, 
i've run composer require for filament and tables 3

what should i do?

**Comment by danharrin** [comment]
Sounds like you have overridden a method on one of your resources and not followed the signature. Try running PHPStan.

**Reply by anindyoy** [reply]
i've tried installing phpstan, but when i run composer, i got the same error
